server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMailboxesCreationProcessor.java

```diff
@@ -19,6 +19,8 @@
 
```

Comment:
I'm using the type system for unescaping (Because there is a relevant place in the code: MailboxFactory). However considering the "escaping" feature, we don't have such places yet. Maybe we need a mailboxPathFactory then.